input/input.go

```diff
@@ -0,0 +1,116 @@
+package input
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type command interface {
+	InOrStdin() io.Reader
+}
+
+func ReadArgs(args []string) ([]string, error) {
+	if !isInputFromPipe() {
+		return args, nil
+	}
+
+	return readPipedArgs()
+}
+
+func isInputFromPipe() bool {
+	fileInfo, _ := os.Stdin.Stat()
+	return fileInfo.Mode()&os.ModeCharDevice == 0
+}
+
+func readPipedArgs() ([]string, error) {
+	var args []string
+
+	input, err := ioutil.ReadAll(os.Stdin)
+	if err != nil {
+		return args, err
+	}
+	args = append(args, strings.Split(string(input), " ")...)
+
+	return args, nil
+}
```

Comment:
Just a thought. As I was looking on making an argument for the cobra guys to accept my possible PR before doing it, I saw that when you pass arguments non-piped, the shell does the expansion for you. So in that case files come already without wildcards. When piping arguments, that's a different story. So we still need the wildcard expansion in the app, which happens in query files, but code will be simpler in general. 

But I think we could remove `ReadArgs` and the 2 functions in there if we say that the way to use a file that is multiline you need to use `xargs`. In that way, we can simply say we don't support piping stdin but there is still a way to use files.

For example
```
$ echo queries/**/*.gql | gql-lint deprecations --schema path/to/schema.gql
$ cat queries_list.txt | gql-lint deprecations --schema path/to/schema.gql
```
would be 
```
$ echo queries/**/*.gql | xargs gql-lint deprecations --schema path/to/schema.gql
$ cat queries_list.txt | xargs gql-lint deprecations --schema path/to/schema.gql
```
BUT, as this is done and supported now, we can simply keep it.

Reply:
much simpler! there it goes https://github.com/sketch-hq/gql-lint/pull/11/commits/8e8998f37b539aafcfc51e1338c1e5e9f1652a0c